gamedata.cpp

```diff
@@ -4978,7 +4978,7 @@ static TerrainType td[] = {
 	 {-1,-1,-1},
 	 0,-1,-1,-1,
 	 0,{-1,-1,-1,-1,-1,-1}},
-	{"ocean", "ocean", "barrenocean", '-', R_OCEAN,
+	{"deadwater", "deadwater", "barrenocean", '-', R_OCEAN,
```

Comment:
This  change is actually already in master.. it will merge cleanly.  Github is just confused because I froze the NO v7 stable branch and am merging from a branch off of that onto master.  The only real change in this commit is the change on line 900 in text_report_generator.cpp